dagster_project/slack/postgres_service.py

```
@@ -3,6 +3,14 @@
 import psycopg2
 from psycopg2.extras import execute_values
 from typing import Dict, Any, List, Optional
+import subprocess
```

Ruff (0.11.9)

`subprocess` imported but unused

Remove unused import: `subprocess`

(F401)